app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::CommentsController < ApplicationController
+  def index
+    @comments = Comment.all
+    render json: @comments
+  end
+
+  def create
+    @comment = Comment.new(
+      text: comment_params[:text],
+      users_id: comment_params[:users_id]
+    )
+    if @comment.save
+      @comment.update_post_comments_counter
+      render json: @comment, status: :created
+    else
+      render json: { errors: @comment.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
+
```

Comment:
![image](https://github.com/Benawi/Blog-App/assets/93039359/67cd85c6-819f-42b9-b98b-327f3f6c1506)

- [ ] It would be nice if you fix the highlihgted link in the Table of Contents to make your readme descriptive and profes